content/docs/details/accessing-aws-resources/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Accessing AWS Services
+publishedDate: '2022-03-23T14:00:00.0Z'
+description: How to configure permissions so that Roadie can access AWS resources in your account.
+---
+## Prerequisites
+
+* The Roadie account ID.
+* The Roadie backend role.
+* (optional) An aws permissions policy name to associate with the role. 
```

Comment:
Should we not explain where to get these things? People might land on this page out of context.

Reply:
I added a pointer to the settings page where they can get these.